### ISSUE TITLE
GetAnimeDetailsForInvalidAnimeId integration test broken

### ISSUE THREAD

**Issue by LHCGreg** [LHCGreg/mal-api#7]
```
MalApi.Integration.GetAnimeDetailsTest.GetAnimeDetailsForInvalidAnimeId:
  Expected: <MalApi.MalAnimeNotFoundException>
  But was:  <MalApi.MalApiRequestException> (Failed getting anime details for anime ID 99999. The remote server returned an error: (404) Not Found.)
   at MalApi.MyAnimeListApi.ProcessRequest[TReturn,TData](HttpWebRequest request, Func`3 processingFunc, TData data, String baseErrorMessage) in c:\Users\Greg\Documents\Programming\MalApi\MalApi\MyAnimeListApi.cs:line 119
   at MalApi.MyAnimeListApi.GetAnimeDetails(Int32 animeId) in c:\Users\Greg\Documents\Programming\MalApi\MalApi\MyAnimeListApi.cs:line 218
   at MalApi.Integration.GetAnimeDetailsTest.<>c__DisplayClass5.<GetAnimeDetailsForInvalidAnimeId>b__1() in c:\Users\Greg\Documents\Programming\MalApi\MalApi.Integration\GetAnimeDetailsTest.cs:line 37
   at NUnit.Framework.Assert.Throws(IResolveConstraint expression, TestDelegate code, String message, Object[] args)
```

**Comment by LHCGreg** [comment]
MAL is now returning a 404 Not Found HTTP status code instead of 200 OK with a page saying the anime wasn't found.

**Comment by LHCGreg** [comment]
Fixed in e3f96962f758798e87088fcd4e86821edb8f4027.
